spyder/config/main.py

```diff
@@ -490,7 +490,7 @@
               'pydocstyle/ignore': '',
               'pydocstyle/match': '(?!test_).*\\.py',
               'pydocstyle/match_dir': '[^\\.].*',
-              'advanced/command_launch': 'pyls',
+              'advanced/module': 'pyls',

```

Comment:
I think this shouldn't be called module, as it may contain other commands beyond Python

Reply:
You mean that we could test things with Microsoft's LSP for Python?

Reply:
For instance, or other executables such as Elixir-LS

Reply:
This change only applies to the Python language server (I didn't touch the dialog for other languages).

The thing is we're running the PyLS with this command:

    sys.executable -m pyls --tcp --host {host} --port {port}

As you can see, users need to provide a Python module (not a Python script or another command) to run the server. That's why our current text in Preferences is misleading and I think it's better to change it.

Of course, this rules out using Microsoft's Python server for now. But we're totally integrated with the PyLS, so trying to support it would require a big refactoring (which could be work for Spyder 5 or even 6, but certainly not for version 4).